jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_CHKPT

```diff
@@ -23,8 +23,8 @@ export GPREFIX="${GDUMP}.t${gcyc}z."
 export APREFIX="${CDUMP}.t${cyc}z."
 
 # Generate COM variables from templates
-RUN=${GDUMP} YMD=${gPDY} HH=${gcyc} generate_com -rx COM_ATMOS_HISTORY:COM_ATMOS_HISTORY_TMPL
-RUN=${GDUMP} YMD=${gPDY} HH=${gcyc} generate_com -rx COM_ATMOS_ANALYSIS:COM_ATMOS_ANALYSIS_TMPL
+RUN=${GDUMP} YMD=${gPDY} HH=${gcyc} generate_com -rx COM_ATMOS_HISTORY
+RUN=${GDUMP} YMD=${gPDY} HH=${gcyc} generate_com -rx COM_ATMOS_ANALYSIS
```

Comment:
This is not the right way to use previous cycle paths.

Reply:
Sorry, must've missed this when these were first added in the previous PR. Rahul is correct; for consistency, the variable name should be `COM_ATMO_HISTORY_PREV`, etc., which means you will also still need to explicitly provide the template.